dsa-solutions/lc-solutions/0000-0099/0039-Combination-Sum.md

```diff
@@ -0,0 +1,194 @@
+---
+id: Combination Sum
```

Comment:
replace `id: Combination Sum` to `id: combination-sum`

Reply:
@PradnyaGaitonde complete your task

Reply:
@Ajay-Dhangar Yes completed it.